cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto

```diff
@@ -161,3 +166,29 @@ message GetPackageRepositoriesResponse {
   // List of PackageRepository
   repeated PackageRepository repositories = 1;
 }
+
+message PackageReference {
+  // required package name should be unique within the scope of specified repository, if specified
+  string name = 1; 
+
+  // A PackageRepository is identified by a fully qualified name and
+  // optionally a specific namespace.
+  message PackageRepositoryReference {
+    string name = 1;
+    string namespace = 2;
+  }
+  // An optional package repository reference where this package is located. Not
+  // all plugins may support this back reference (eg. kapp-controller)
+  PackageRepositoryReference repository = 2;
+}
+
+message GetPackageMetaRequest {
+  PackageReference package = 1;
+}
+
+message GetPackageMetaResponse {
+  message PackageMeta {
+    string readme = 1;
```

Comment:
I was thinking about this field - Helm charts have a readme, but a more general option might be `longDescription`, noting that it can be markdown, but we can discuss that in the design doc soon.

Reply:
Michael, this is not ready for review in any shape. Its a draft just to see what it costs to make the "electrons flow". When I am ready for review the interface will be much more generic and not look like that. So no need to worry

Reply:
Ah sorry... I thought you'd created a draft PR because it wasn't ready for review but wanted some feedback (something I often do). No probs, I'll not look at something until you request it :)